cmd/cmd.go

```diff
@@ -1157,7 +1158,7 @@ func RunServer(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	err = server.Serve(ln)
+	err = server.Serve(build.LibEmbed, ln)
```

Comment:
Do we need to bring this all the way up to `cmd`?

Reply:
Good point - remnant from the top-level layout.  I'll push that down into server.